oasislmf/model_preparation/sum_inputs.py

```diff
@@ -231,32 +177,24 @@ def write_mapping_file(sum_inputs_df, target_dir):
     return sum_mapping_fp
 
 
-
-@oasis_log
 def get_column_selection(summary_set):
     # Summary Set Not defined in settings file
     if "oed_fields" not in summary_set.keys():
```

Comment:
As with Python dicts, you can check of the existence of a key in a dict by just using the dict, and not the dict keys e.g. 

    if "oed_fields" not in summary_set:

is perfectly fine here